packages/documentation-framework/layouts/sideNavLayout/sideNavLayout.js

```diff
@@ -209,20 +210,31 @@ export const SideNavLayout = ({ children, groupedRoutes, navOpen: navOpenProp })
   );
 
   const Header = (
-    <PageHeader
-      className="ws-page-header"
-      headerTools={(algolia || hasVersionSwitcher) && <HeaderTools
-        versions={versions}
-        hasSearch={algolia}
-        hasVersionSwitcher={hasVersionSwitcher}
-        hasDarkThemeSwitcher={hasDarkThemeSwitcher}
-        pathPrefix={pathPrefix} />}
-      logo={prnum ? `PR #${prnum}` : <Brand src={logo} alt="Patternfly Logo" />}
-      logoProps={{ href: prurl || pathPrefix || '/' }}
-      showNavToggle
-      topNav={topNavItems.length > 0 && <TopNav navItems={topNavItems} />}
-    />
-  );
+    <Masthead className="ws-masthead">
+      <MastheadToggle>
+        <PageToggleButton variant="plain" aria-label="Global navigation">
+          <BarsIcon />
+        </PageToggleButton>
+      </MastheadToggle>
+      <MastheadMain>
+        <MastheadBrand href={prurl || pathPrefix || '/'}>
+          {prnum ? `PR #${prnum}` : <Brand src={logo} alt="Patternfly Logo" />}
```

Comment:
It may be worth considering using the logo from core, it's bigger and looks better IMO - https://github.com/patternfly/patternfly/blob/5eee883c9aadfa4834798de2ee3c00dfa8a37a6a/src/patternfly/demos/Masthead/masthead-template.hbs#L15-L20

Reply:
Do you know offhand the best way to do that? Is it already something I can import and use or should I copy and paste an svg from core here?

Reply:
On org anyways, the core demo images are at `/assets/images/...`. So they're already there somewhere :D 

I'm not sure if there is a best practice around `import`ing the file and referencing it like most of the react examples do, but you could just do it like this which is pretty simple

```
var logoMd = '/assets/images/logo__pf--reverse-on-md.svg';
var logo = '/assets/images/logo__pf--reverse--base.svg';
var logoBase = '/assets/images/logo__pf--reverse--base.png';

<Brand src={logoBase} alt="PatternFly logo" widths={{ default: '180px', md: '220px' }}>
  <source media="(min-width: 768px)" srcSet={logoMd} />
  <source srcSet={logo} />
 </Brand>
```